embedded-graphics/src/lib.rs

```diff
@@ -14,7 +14,7 @@
 //! * [BMP images - `tinybmp`](https://crates.io/crates/tinybmp)
 //! * [TGA images - `tinytga`](https://crates.io/crates/tinytga)
 //! * [ProFont monospace font - `profont`](https://crates.io/crates/profont)
-//! * [Picofont Pico8 font - `picofont`](https://crates.io/crates/picofont)
+//! * [Picofont Pico8 font - `picofont`](https://crates.io/crates/embedded_picofont)
```

Comment:
There's also a corresponding line in `embedded-graphics/README.md` that needs this fix too.